cccatalog-api/cccatalog/api/views/image_views.py

```diff
@@ -271,6 +271,14 @@ def get(self, request, identifier, format=None, view_count=0):
 
         return resp
 
+    @swagger_auto_schema(operation_id="image_delete")
+    def delete(self, request, identifier, format=None):
+        image = Image.objects.get(identifier=identifier)
+        image.delete()
+        content = {'status': 'NO CONTENT'}
```

Comment:
Just a tiny nitpick - you don't need to return any JSON body when you use HTTP 204. Everything else looks good so far.